public/static/app/common/MuteDataService.js

```diff
@@ -0,0 +1,81 @@
+mciModule.factory('MuteDataService', function(
+  $log, Stitch, STITCH_CONFIG
+) {
+  // Get the mutes as a promise
+  // :param query: The query to evaluate.
+  // :param projection: Specify the fields in the returned documents.
+  const queryQ = (query, projection) => {
+    return Stitch.use(STITCH_CONFIG.PERF).query(function (db) {
+      // Remove Undefined values, null and false are acceptable.
```

Comment:
This comment is a little confusing. It is difficult to tell what is being removed and what is acceptable. Maybe break it into two sentences?

(same for this comment in other places in this file)

Reply:
Actually this comment in this location was left over when some some code was removed,

I'll update the comments where it should make sense (couple with an omit).